packages/itwinui-css/src/slider/slider.scss

```diff
@@ -88,6 +122,13 @@ $tick-height: $iui-baseline; // 11px
   @include themed {
     background-color: t(iui-color-foreground-primary);
   }
+
+  @at-root .iui-vertical & {
+    width: $track-height;
+    bottom: 0;
```

Comment:
Maybe in order to avoid constantly overriding horizontal rule we could have `iui-horizontal` and `iui-vertical` like we have in Tabs.

Reply:
Thanks, I went through `tabs.scss`, trying to do something similar in `sliders.scss` now

Reply:
Hm, I was thinking to kind of keeping the same as it was where common styling is left but there would be `iui-horizontal` and `iui-vertical` modifiers that have orientation specific rules.

Reply:
@bentleyvk Ohh, yes, maybe I misunderstood. Could you please give a small code sample to help me understand what you meant? Also, just curious, do you feel the newer version is more cleaner and better? Or do you prefer the old version?

Reply:
I tried implementing the changes that you suggested. Kindly take a look and let me know if you suggest any other changes.

Reply:
**[DONE]** Marking it as done as Vilius approved this PR. @bentleyvk, can you please mark this as resolved? I don't want to do it myself in case you wanted to recommend some more changes. Thanks.